packages/itwinui-css/src/bridge.scss

```diff
@@ -255,9 +295,9 @@
     box-shadow: var(--stratakit-shadow-surface-xl);
   }
 
+  // Menu item hover state background color
   .iui-list-item {
-    // Disabling hover to keep the code simple
-    --_iui-list-item-background-hover: transparent;
+    --_iui-list-item-background-hover: var(--stratakit-color-bg-glow-on-surface-neutral-hover);
   }
```

Comment:
Isn't the global mapping the same?

```suggestion
```

Reply:
Ah it's using `--iui-color-background-hover` instead of `--iui-color-background-transparent-hover`. Probably something we should fix in the base component?

Reply:
For reference, here is the line setting `.iui-list-item`'s `background-color` on `:hover`: https://github.com/iTwin/iTwinUI/blob/e6ba549c15ebb375c7b0ec550b957e847a5bd21c/packages/itwinui-css/src/menu/menu.scss#L36

PR opened: https://github.com/iTwin/iTwinUI/pull/2641